quelpa.el

```diff
@@ -1689,21 +1721,25 @@ If t, `quelpa' tries to do an upgrade.
 
 :stable
 
-If t, `quelpa' tries building the stable version of a package."
+If t, `quelpa' tries building the stable version of a package.
+
+:force
+
+If t, `quelpa' tries to do an upgrade regardless state of local package repo."
```

Comment:
Please see earlier comment about the variable name.  If the primary purpose of this option is to discard local, uncommitted changes, I think we should try to describe that more explicitly.